package.xml.tpl

```diff
@@ -38,7 +38,7 @@
             <file role="php" baseinstalldir="gherkin" name="vendor/.composer/autoload.php" />
             <file role="php" baseinstalldir="gherkin" name="vendor/.composer/autoload_namespaces.php" />
             <file role="php" baseinstalldir="gherkin" name="phpdoc.ini.dist" />
-            <file role="php" baseinstalldir="gherkin" name="phpunit.xml.dist" />
+            <file role="php" baseinstalldir="gherkin" name="phpunit.dist.xml" />
```

Comment:
Is this `package.xml.tpl` actually used for anything? It looks quite outdated (minimum php = 5.3.1, for example).